locales/it/index.yml

```diff
@@ -1267,6 +1273,9 @@ faq:
   57:
     title: "Hai avuto problemi durante un pagamento?"
     content: !include faq/faq57.md
+  58:
```

Comment:
Is this faq used?

Reply:
removed within https://github.com/pagopa/io-app/pull/1889/commits/76645550aebcadbc662c5c5d7c94ad96a2962b94. It was used to represent the idp-related content as part of the faq, now it is the main content displayed.